instapy/instapy.py

```diff
@@ -2354,14 +2360,7 @@ def interact_by_users_tagged_posts(self,
 
                                 if self.use_clarifai and commenting:
                                     try:
-                                        checked_img, temp_comments = (
-                                            check_image(self.browser,
-                                                        self.clarifai_api_key,
-                                                        self.clarifai_img_tags,
-                                                        self.clarifai_img_tags_skip,
-                                                        self.logger,
-                                                        self.clarifai_full_match))
-
+                                        checked_img, temp_comments, clarifai_tags = (self.query_clarifai())
```

Comment:
What is the reason for the outer parenthesis?

```suggestion
                                        checked_img, temp_comments, clarifai_tags = self.query_clarifai()
```

Reply:
just a copy paste ...

Reply:
fixed